### PR TITLE
Remove references to grunt

### PR DESCRIPTION
**Pull request by megawac**
As far as I can tell grunt is no longer used

### CODE REVIEW

**Comment by davidchambers** [comment]
Thanks!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -14,7 +14,7 @@
     commit message, wrapped at 72 characters. Do not commit changes to
     __dist/ramda.js__.
 
-5.  Run `make test lint`, (or `grunt test`) and address any errors. Preferably,
+5.  Run unit tests by `make test lint` and address any errors. Preferably,
```
Comment (davidchambers) [inline CONTRIBUTING.md]: This doesn't read well to me. I'd either remove the _unit tests by_ or begin with _Run unit tests by running_.

Comment (buzzdecafe) [inline CONTRIBUTING.md]: should run `npm test` here anyways, since that will rebuild and restore dist

**Comment by davidchambers** [comment]
:deciduous_tree:

**Comment by CrossEye** [comment]
I believe you're right.  Grunt is no longer used in the main repo.  (It's still is used in docs, though.)

@buzzdecafe, was there anything left to do in this effort?

**Pull request merged**